webapp/app/templates/components.html

```diff
@@ -154,8 +154,7 @@ <h5 class="mb-0">{{ card['heading'] }}</h5>
 {% macro details(title, paragraphs, list_items=None, details_id='') %}
     <div id="details-{{ details_id }}" class="details-card ml-0 pl-0">
         <div class="card">
-            <div class="card-header unstyled-card-header d-sm-flex justify-content-between align-items-center cursor-pointer mb-0"
-                 id="heading-card_id">
+            <div class="card-header unstyled-card-header d-sm-flex justify-content-between align-items-center cursor-pointer mb-0">
```

Comment:
maybe we could just use `id="heading-card-id-{{ details_id }}` or do we not need this id for anything else?

Reply:
We don't use this id anywhere^^